Formula/llvm.rb

```diff
@@ -254,8 +311,21 @@ def caveats
           return 0;
         }
       EOS
-      system "#{bin}/clang++", "test.cpp", "-o", "test"
-      system "./test"
+
+      ohai "#{ENV.cflags}"
+
+      if MacOS::CLT.installed?
+        # Test with CLT
+        system "#{bin}/clang++", "-v", "-std=c++11", "-stdlib=libc++", "test.cpp", "-o", "test"
+        system "./test"
+      end
+
+      if build.with? "libcxx" || ! MacOS::CLT.installed?
+        # Test with libcxx
```

Comment:
I removed this and other (formula-structure) comments. I completely disagree with this request but it is much more important for me to have fully-featured and working `llvm 3.8.0` formula than the comments. I think not having comments and appropriately called variables will make work of future maintainers of the formula much more difficult.